src/smart-components/request/requests.js

```diff
@@ -44,7 +44,11 @@ const Requests = ({ fetchRequests, isLoading, pagination, history }) => {
       postMethod={ fetchData }/> } />
   </Fragment>;
 
-  const areActionsDisabled = (requestData) => requestData && requestData.state ? !isRequestStateActive(requestData.state.title) : true;
+  const areActionsDisabled = (requestData) => {
+    console.log('DEBUG - requestData: ', requestData);
```

Comment:
console.log

Reply:
thanks, forgot to push yesterday after I removed the line.